code128.py

```diff
@@ -0,0 +1,66 @@
+# taken from https://bitbucket.org/whitie/python-barcode
+
```

Comment:
We probably want to note that, at the time you grabbed this source, it had an MIT license. We don't want trouble down the road if they re-license the project - it should be crystal-clear to whoever is reading this that we have permission to use it.